src/dropdown/Dropdown.tsx

```diff
@@ -28,6 +26,8 @@ import Spinner from "../spinner/Spinner";
 import {SINGLE_ALPHANUMERIC_CHARACTER_REGEX} from "../core/utils/stringConstants";
 import useOnClickOutside from "../core/utils/hooks/onClickOutside";
 
+import "./_dropdown.scss";
```

Comment:
I believe we should add a comment here explaining why it is imported here to prevent ambiguity